src/msg/async/Event.cc

```diff
@@ -414,7 +415,7 @@ int EventCenter::process_events(unsigned timeout_microseconds,  ceph::timespan *
   bool blocking = pollers.empty() && !external_num_events.load();
   if (!blocking)
     timeout_microseconds = 0;
-  tv.tv_sec = timeout_microseconds / 1000000;
+  tv.tv_sec = div_round_up(timeout_microseconds, 1000000);
```

Comment:
This seems wrong to me: if `timeout_microseconds` is 1, wouldn't both `tv_sec` and `tv_usec` be set to 1, meaning that a 1 microsecond timeout would suddenly become a 1 second timeout?

Given changes to `EventEpoll.cc` and `EventPoll.cc`, it doesn't seem like any changes to `Event.cc` are needed.

Reply:
Yes, you're right. This commit is wrong, there's no rounding error here. This mistake never had any effect in my testing because there were no sub-second timers.